suites/tentacle/rados/tier-3_rados_test_6+2_ecpools.yaml

```diff
@@ -0,0 +1,597 @@
+      module: pool_tests.py
+      polarion-id: CEPH-83573424
+      config:
+        verify_pool_target_ratio:
+          configurations:
+            pool-1:
+              profile_name: ec62_10
+              pool_name: ec62_pool10
+              pool_type: erasure
+              pg_num: 32
+              k: 6
+              m: 2
+              plugin: isa
+              crush-failure-domain: host
+              target_size_ratio: 0.8
+              max_objs: 300
+              rados_read_duration: 10
+              delete_pool: true
+      desc: Specifying pool bounds on pool Pgs - Verify target_size_ratio
+
+  - test:
+      name: Autoscaler test - pool pg_num_min
+      module: pool_tests.py
+      polarion-id: CEPH-83573425
+      config:
+        verify_pg_num_min:
+          configurations:
+            pool-1:
+              profile_name: ec62_11
+              pool_name: ec62_pool11
+              pool_type: erasure
+              k: 6
+              m: 2
+              plugin: isa
+              crush-failure-domain: host
+              pg_num_min: 16
+              max_objs: 300
+              rados_read_duration: 10
+              delete_pool: true
+      desc: Specifying pool bounds on pool Pgs - Verify pg_num_min
+
+  - test:
+      name: client pg access
+      module: test_client_pg_access.py
+      polarion-id: CEPH-83571713
+      config:
+        verify_client_pg_access:
+          num_snapshots: 20
+          num_objects: 250
+          configurations:
+            pool-1:
+              profile_name: ec86_12
+              pool_name: ec86_pool12
+              pool_type: erasure
+              k: 6
+              m: 2
+              plugin: isa
+              crush-failure-domain: host
+              disable_pg_autoscale: true
+      desc: many clients clients accessing same PG with bluestore as backend
+
+  - test:
+      name: Migrate data bw pools.
+      module: test_data_migration_bw_pools.py
+      polarion-id: CEPH-83574768
+      config:
+        pool-1-type: replicated
+        pool-2-type: erasure
+        pool-1-conf: sample-pool-1
+        pool-2-conf: sample-pool-3
+        pool_configs_path: "conf/tentacle/rados/test-confs/pool-configurations.yaml"
+      desc: Migrating data between different pools. Scenario-2. RE -> EC
+
+  - test:
+        name: Migrate data bw pools.
+        module: test_data_migration_bw_pools.py
+        polarion-id: CEPH-83574768
+        config:
+          pool-1-type: erasure
+          pool-2-type: replicated
+          pool-1-conf: sample-pool-3
+          pool-2-conf: sample-pool-3
+          pool_configs_path: "conf/tentacle/rados/test-confs/pool-configurations.yaml"
+        desc: Migrating data between different pools. Scenario-3. EC -> RE
+
+  - test:
+      name: Migrate data bw pools.
+      module: test_data_migration_bw_pools.py
+      polarion-id: CEPH-83574768
+      config:
+        pool-1-type: erasure
+        pool-2-type: erasure
+        pool-1-conf: sample-pool-4
```

Comment:
sample-pool-3 & 4 are  MSR EC pools. update the config file with new configs and update the suite file.